test/unit/utils/bitcoin.test.ts

```diff
@@ -23,12 +23,4 @@ describe("Bitcoin", () => {
             p2wpkhv0: "0xe8df018c7e326cc253faac7e46cdc51e68542c42",
         });
     });
-
-    it("should correctly decode & encode address", () => {
```

Comment:
Why don't we need this anymore?

Reply:
It cannot be run as a unit test because when `btcAddressFromParams` attempts to construct a `BitcoinAddress` on an empty `registry`, it will fail because no metadata is set. This functionality is already tested in the replace integration tests though.

Reply:
It cannot be run as a unit test because when btcAddressFromParams attempts to construct a BitcoinAddress on an empty registry, it will fail because no metadata is set. This functionality is already tested in the replace integration tests though.